src/nvim/fileio.c

```diff
@@ -6558,16 +6558,17 @@ aucmd_prepbuf (
     /* Split the current window, put the aucmd_win in the upper half.
      * We don't want the BufEnter or WinEnter autocommands. */
     block_autocmds();
-    make_snapshot(SNAP_AUCMD_IDX);
+    /* make_snapshot(SNAP_AUCMD_IDX); */
     save_ea = p_ea;
     p_ea = false;
 
     /* Prevent chdir() call in win_enter_ext(), through do_autochdir(). */
     save_acd = p_acd;
     p_acd = false;
 
-    (void)win_split_ins(0, WSP_TOP, aucmd_win, 0);
-    (void)win_comp_pos();       /* recompute window positions */
+    win_enter(aucmd_win, false);
```

Comment:
if the aucmd window is not new `win_append` is needed here, to match the `win_remove`. Also I don't think `win_enter` should be used (it is only to change the real current window).